### DISCUSSION TITLE
Cancelling payments for account where I no longer have email access

### DISCUSSION THREAD

**Discussion by HughParry** [community/community#58587] [New to GitHub]
I have a copilot subscription related to an account that I no longer have email access to. How do I cancel it? It's charged me $10 per month for a few months now.

I've reached out to github support and got no answer

I don't want my bank to reject all payments from github, as I'm pretty happy with copilot, and don't want to cancel it for this account which i have access to.

How can I resolve this?

**Comment by queenofcorgis** [comment]
Glad to hear you're enjoying GitHub Copilot. What is your ticket number? Support are the best folks to go to in this instance and I can make sure it went to the proper place 😄 .

**Reply by Longisuaw** [reply]
Hi, my ticket is opened for over 2 weeks now and still no response. Please help @queenofcorgis #3411137

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by zagranovskyi** [comment]
Hello! I have a similar issue, could you please provide the proper way to adress cancelling payments for account where I no longer have email access?
